packages/teams-js/README.md

```diff
@@ -1,24 +1,12 @@
 # Microsoft Teams JavaScript client SDK
 
-Welcome to the Teams client SDK monorepo! For breaking changes, please refer to our changelog in the monorepo root of the `2.0-preview` branch.
-
-This JavaScript library is part of the [Microsoft Teams developer platform](https://developer.microsoft.com/microsoft-teams/). See full [SDK reference documentation](https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client).
```

Comment:
We should leave this sentence in. This is the text that will be included on the npm site for teams-js and I think having links to the documentation here is important. Please make sure it links to the 2.0 version of the SDK docs.